### ISSUE TITLE
install fails after autoreconf -fi

### ISSUE THREAD

**Issue by flyingblindonarocketcycle** [mikebrady/shairport-sync#746]
Hi

I realize that this issue has been raised and closed already but it appears to be closed without resolution.
I have a Raspberry pi 3 and 3 raspberry pi zero W's.  I have installed shairport-sync on all of them and my house JAMS.  One of my Pi Zero W's had a flash card die and as I fallow the install instructions it fails after autoreconf -fi.

```
pi@balcony:~/shairport-sync $ autoreconf -fi
autom4te: cannot create autom4te.cache: No such file or directory
aclocal: error: echo failed with exit status: 1
autoreconf: aclocal failed with exit status: 1
```
Any ideas?

**Comment by flyingblindonarocketcycle** [comment]
well this is embarrassing.   It appears I need to put a sudo before all the instructions.

**Comment by flyingblindonarocketcycle** [comment]
My install "appears" to be successful.  The "balcony" shows up now as one of my audio zones but i have yet to get any audio from it.  Do I no longer need an asound.conf file with this install?  Also my other pi's allow me to use this command 
`sudo -u shairport-sync alsamixer -D equal `
but this one tells me
pi@balcony:~ $ sudo -u shairport-sync alsamixer -D equal
Failed to open controls file:/home/shairport-sync/.alsaequal.bin.
cannot open mixer: Operation not permitted

My other installs were all done using the raspberry gui for add/remove programs.  maybe I well wipe this thing again and try that way.  Not sure what to do.

**Comment by mikebrady** [comment]
Thanks for the post. It's good to hear that you're making good use of Shairport Sync!

It really does sound like an issue with permissions, so perhaps starting over, with a fresh copy of Raspbian, would be worth while. I would suggest avoiding the GUI for installing/removing libraries -- instead, I'd suggest following the guide at [INSTALL.md](https://github.com/mikebrady/shairport-sync/blob/master/INSTALL.md). The GUI interface has a Terminal application, right?

It's very important to take note of the  standard convention -- in the guide, if a command has the `#` prompt, you must be in the superuser mode, and if the command in the guide has the `$` prompt, you must be in a regular user mode. It's not really okay to be in the superuser mode all the time -- that doesn't end well :)

If you make a mistake in this, then problems like you have encountered can occur. For example, if you run `make` in the superuser mode and then later on try to use `make` in regular user mode, it will fail because it will not be able to overwrite files written while in the previous superuser mode.

So perhaps the best thing is to start over with a fresh flashing of Raspbian...

**Comment by flyingblindonarocketcycle** [comment]
Configuring things so that shairport-sync will work with alsa eq has always been confusing to me.  I did get that to work in the past where I absolutely HAD to have eq but this current build doesn't need it.

I changed my asound.conf file to a simple config and now this latest pi zero w is a functioning audio zone once again.

It would be nice to understand the eq issue but as for the install, it works.  

Thanks for your work on this Mike.
